db/migrate/20160619064544_change_memory_reserve_data_type_in_vm.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeMemoryReserveDataTypeInVm < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :memory_reserve, :bigint
```

Comment:
I looked at the Disk table, since the sizes will be the same. It has bigint for size. The tests fail if you leave it as integer, and the tests are built from fixtures with real world data. (Also it failed on my own system with just a 1G disk).
So I went with the same type defined in the "disks" table.